apps/web/.env.example

```diff
@@ -11,6 +11,10 @@
 # This secret is provided by the Azure AD during app registration.
 # AUTH_CLIENT_SECRET=
 
+# The unique application (api) uuid assigned to the web app by Azure AD when
+# the app was registered.
+# AUTH_API_CLIENT_ID=
```

Comment:
to work locally the web app needs this value set in the env.local

Reply:
unless we disable it locally for both WEB and API as we do for Auth

Reply:
@francescobrenci-AR  I can't see this file